ProjectLighthouse/Pages/SlotPage.cshtml

```diff
@@ -54,6 +57,36 @@
         </div>
     </div>
 </div>
+<div class="ui yellow segment">
+    <h1>Comments</h1>
+    @if (Model.Comments.Count == 0)
+    {
+        <p>There are no comments.</p>
+    }
+
+    @foreach (Comment comment in Model.Comments!)
+    {
+        DateTimeOffset timestamp = DateTimeOffset.FromUnixTimeSeconds(comment.Timestamp / 1000);
+        StringWriter messageWriter = new();
+        HttpUtility.HtmlDecode(comment.getComment(), messageWriter);
```

Comment:
This doesnt allow XSS does it?

Reply:
It shouldn't, it's straight from the user class
![test](https://i.imgur.com/jOSnSIx.png)